reactor-netty-http-brave/src/main/java/reactor/netty/http/server/DelegatingHttpServerOperations.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2011-Present VMware, Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package reactor.netty.http.server;
```

Comment:
this is an interesting technique, but isn't it incompatible with java modules ? (shared package between automatic module `reactor.netty.http` and `reactor.netty.http.brave`)